api/models/wallet.model.js

```diff
@@ -18,6 +18,7 @@ const walletSchema = new Schema({
 					isIncome: { type: Boolean, required: true },
 					title: String,
 					description: String,
+					isShowReport: Boolean,
```

Comment:
```suggestion
					isShowReport: { type: Boolean, default: true },
```
mình thiếu report khúc ni nè, để lỡ mà họ thêm expense thì mặc định là tính vô report á. Có thể hok ảnh hưởng mà mình làm cho cái db hắn đồng bộ với nhau cho hắn tốt hey